src/nvim/ex_docmd.c

```diff
@@ -8968,6 +8969,16 @@ makeopens (
     if (nr > 1 && ses_winsizes(fd, restore_size, tab_firstwin) == FAIL)
       return FAIL;
 
+    // Take care of tab-local working directories if applicable
+    if (tp->tp_localdir) {
+      if (fputs("if has('nvim') | tcd ", fd) < 0
+          || ses_put_fname(fd, tp->tp_localdir, &ssop_flags) == FAIL
+          || fputs(" | endif", fd) < 0
+          || put_eol(fd) == FAIL) {
+           return FAIL;
```

Comment:
Looks like something too indented: 2-space indent should align `return` with space after `if`.